saq/queue/postgres.py

```diff
@@ -484,7 +484,13 @@ async def iter_jobs(
 
     async def abort(self, job: Job, error: str, ttl: float = 5) -> None:
         job.error = error
```

Comment:
you don't need to set this anymore right, you can pass error to update?

Reply:
You're right. Updated.